### ISSUE TITLE
  .instance_segmentation_frame return nothing

### ISSUE THREAD

**Issue by mariiak2021** [allenai/ai2thor#867]
Hi all,

I have Controller like this:
controller = Controller(
    agentMode="default",
    visibilityDistance=1.5,
    scene="FloorPlan213",

    # step sizes
    gridSize=0.25,
    snapToGrid=True,
    rotateStepDegrees=90,

    # image modalities
    renderDepthImage=True,
    renderInstanceSegmentation=True,

    # camera properties
    width=1250,
    height=600,
)

And trying to see Instance segmentation frames:
    print(event.instance_segmentation_frame)
    print(event.instance_masks)

But it returns: 
None

Can you please help me to understand, what is missing? @mattdeitke 
Thank you!

**Comment by mattdeitke** [comment]
Hmm.. Looks like it's working for me: https://colab.research.google.com/drive/1o5OTdeQqYs_xMFlmqA7vsr-cR5M-H6Pb?usp=sharing.

Can you try

```python
from ai2thor.controller import Controller
from PIL import Image

controller = Controller(
    agentMode="default",
    visibilityDistance=1.5,
    scene="FloorPlan213",

    # step sizes
    gridSize=0.25,
    snapToGrid=True,
    rotateStepDegrees=90,

    # image modalities
    renderDepthImage=True,
    renderInstanceSegmentation=True,

    # camera properties
    width=1250,
    height=600,
)

Image.fromarray(controller.last_event.instance_segmentation_frame).save("instance_frame.png")
```

**Comment by mariiak2021** [comment]
@mattdeitke Hi!

I returns this:
(ai2thor) mkhan@pascal0:~/ai2thor$ python test.py
Found path: /home/mkhan/.ai2thor/releases/thor-Linux64-a4bcfa890c748c0e9e04de9dc2062bb6a5d1bf3e/thor-Linux64-a4bcfa890c748c0e9e04de9dc2062bb6a5d1bf3e
Traceback (most recent call last):
  File "/dstore/home/mkhan/ai2thor/test.py", line 23, in <module>
    Image.fromarray(controller.last_event.instance_segmentation_frame).save("instance_frame.png")
  File "/home/mkhan/anaconda3/envs/ai2thor/lib/python3.9/site-packages/PIL/Image.py", line 2818, in fromarray
    arr = obj.__array_interface__
AttributeError: 'NoneType' object has no attribute '__array_interface__'


So there is nothing to show - it's empty. :(
Any ideas of what's wrong?

**Comment by mariiak2021** [comment]
@mattdeitke 
I'm using conda to handle ai2thor by the way, if it's matter:
ai2thor                   2.6.2              pyh44b312d_0    conda-forge

**Comment by mattdeitke** [comment]
Ahh, I see. I was able to reproduce this here: https://colab.research.google.com/drive/1o5OTdeQqYs_xMFlmqA7vsr-cR5M-H6Pb?usp=sharing.

Are you able to upgrade to the latest version of AI2-THOR? It looks like this has since been solved.

**Comment by mariiak2021** [comment]
Hi @mattdeitke,

I'm not able to update to the latest version of Ai2-Thor with conda-forge - it's the last available version 2.6.2. :(

If I update using pip, I have troubles with graphical output, as I'm using server with Linux on my Windows laptop. But surprisingly with conda-forge version there are no problems like this (graphical output is working fine).

Anyway I solved the problem with segmentation by adding     "renderObjectImage=True" to the Controller initialization part. I found that this name was used instead of     "renderInstanceSegmentation=True" in versions 2.x of Ai2-Thor.
Then things magically worked. :) For now it should be fine. 

Thank you!
